src/test/java/uk/gov/functional/CleanDatabaseRule.java

```diff
@@ -1,15 +1,10 @@
 package uk.gov.functional;
 
 import org.junit.rules.ExternalResource;
-import uk.gov.functional.db.TestEntriesDAO;
 
-public class CleanDatabaseRule extends ExternalResource {
-    public final TestEntriesDAO testEntriesDAO;
-
-    public CleanDatabaseRule(TestEntriesDAO testEntriesDAO) {
-        this.testEntriesDAO = testEntriesDAO;
-    }
+import static uk.gov.functional.DBHandle.*;
```

Comment:
what's going on here? Is this related to the CT stuff or should it be a separate commit with its own explanation?